src/main/kotlin/no/nav/omsorgspenger/App.kt

```diff
@@ -81,7 +82,7 @@ internal fun Application.omsorgspengerRammemeldinger(applicationContext: Applica
     routing {
         HealthRoute(healthService = applicationContext.healthService)
         OverføringerApi() // todo: autentisering
-        AleneOmApi() // todo: autentisering
+        AleneOmApi(applicationContext.aleneOmOmsorgenService) // todo: autentisering
```

Comment:
Gjelder forsåvidt ikke bare her, men bl.a. her, å bruke named parameters fremfor å basere det på rekkefølgen (the java-way) - så blir det mer leselig i samme slengen.